File: Javascript/8-Kyu/Thinkful-logic-drills-traffic-light.js
// Thinkful - Logic Drills: Traffic light
function updateLight(current) {
  
    return current === 'green' ? 'yellow'
         : current === 'yellow' ? 'red'
         : 'green';
  
};

// Tests
console.log(updateLight('green'));  // --> 'yellow'
console.log(updateLight('yellow'));  // --> 'red'
console.log(updateLight('red'));  // --> 'green'

File: Python/8-Kyu/Thinkful-logic-drills-traffic-light.py
# Thinkful-logic drills: traffic light
def update_light(current):
    
    lights = { 'green' : 'yellow', 'yellow' : 'red', 'red' : 'green'}

    return lights[current]

# Tests
print(update_light('green'))  # --> 'yellow'
print(update_light('yellow'))  # --> 'red'
print(update_light('red'))  # --> 'green'
